src/lib/rest/RestService.cpp

```diff
@@ -475,7 +486,22 @@ std::string restService(ConnectionInfo* ciP, RestService* serviceV)
     commonFilters(ciP, &parseData, &serviceV[ix]);
     scopeFilter(ciP, &parseData, &serviceV[ix]);
 
+
     std::string response = serviceV[ix].treat(ciP, components, compV, &parseData);
+
+    //
+    // If we have gotten this far, and the status is 200, the Input is OK.
```

Comment:
I understand that this idea works as long as a badInput never ends with a ciP->httpStatusCode == SccOk. I guess that we can ensure that (except bug, of course). Is my interpretaiton correct?